### PR TITLE
Adding more UTs - these are basic UTs which will be enhanced

### PR DESCRIPTION
**Pull request by eladiw**
This commit increase the coverage rate to ~56%

### CODE REVIEW

File: build/asset-manifest.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
-  "main.css": "static/css/main.2f893525.css",
```
Comment (morsh) [inline build/asset-manifest.json]: No need to add build files - they will be added automatically when pushed to master

File: client/src/tests/stores/Connection.test.ts [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+  it ('Testing SetupActions', (done) => {
+
+    ConnectionsStore.listen((state) => {
+      expect(state).toHaveProperty('connections');
+      done();
+    });
+    ConnectionsActions.updateConnection('Conn name', {});
+  });
+
+  // it ('Testing SetupActions - failure message', (done) => {
```
Comment (morsh) [inline client/src/tests/stores/Connection.test.ts]: Please remove unnecessary lines

File: client/src/tests/stores/Setup.test.ts [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+describe('Data Source: Samples', () => {
+
+  this.timeout = 10000;
+  beforeAll(() => {
+  })
+
+  it('Testing SetupActions', (done) => {
+
+    let checkState = (state) => {
+      expect(state).toHaveProperty('admins');
```
Comment (morsh) [inline client/src/tests/stores/Setup.test.ts]: Is it possible to compare it to an object instead (or better yet a snapshot - wouldn't require almost any work, and would automatically save your object "image" to compare it later)

Comment (eladiw) [inline client/src/tests/stores/Setup.test.ts]: @morsh Done

**Comment by eladiw** [comment]
@morsh , also I've updated some travis files and git ignore file to make the solution cleaner...

**Pull request merged**